### DISCUSSION TITLE
Transfer profiles without Firefox sync

### DISCUSSION THREAD

**Discussion by 2whlrider** [Floorp-Projects/Floorp#1168] [Feature Request]
I would like to be able to transfer profile information such as bookmarks, passwords, browser settings, autocomplete info, etc. between instances of Floorp and without using Firefox sync. I don't want to share this info with Mozilla.

**Comment by joshika39** [comment]
Hello @2whlrider !

I'm glad to hear these requests! We are currently working in a solution which solves exactly that.

You will be able to sync everything all the way down to your customization settings up to your Google drive or a selfhosted storage.

I'll notify you if there is anything you would need to know!

Thanks

**Reply by alkeryn** [reply]
any news on this ? :)
